source/release-notes/v1.6-release-notes.rst

```diff
@@ -0,0 +1,178 @@
+.. _v1.6-release-notes:
+
+v1.6 Release Notes
+==================
+
+Highlights in 1.6:
+
+- `OnDemand passed a security audit!`_
+- `Added job array support for LSF and PBSPro adapters`_
+- `Added L18n hooks to the Dashboard and Job Composer`_
+- `Added rendering of HTML or Markdown in Job Composer template manifests`_
+- `Changed ngnix_stage to always remove stale PID files`_
+- `Fixed Active Jobs to show empty cell if account is nil`_
+- `Fixed crash in nginx_stage`_
+- `Fixed Job Composer layout when job name is long`_
+- `Fixed Time Used format for Active Jobs`_
+- `Fixed warning about eager loading gems`_
+- `Improved Grid Engine adapter`_
+- `Improved Slurm adapter`_
+- `Updated web-application dependencies`_
```

Comment:
We update dependencies every time so it doesn't need to be explicitly called out, unless its significant dependency like Passenger version or default Ruby version.